src/content/api/node.mdx

```diff
@@ -223,7 +224,7 @@ stats.toJson({
 All available options and presets are described in the stats [documentation](/configuration/stats).
 
 > Here’s an [example]
-(https://github.com/webpack/analyse/blob/master/app/pages/upload/example.json)
+(https://github.com/webpack/analyse/blob/master/app/pages/upload/example1.json)
```

Comment:
```suggestion
(https://raw.githubusercontent.com/webpack/analyse/master/app/pages/upload/example1.json)
```

Reply:
With link to raw content you can't go anywhere but the json file, maybe you wanna check around then? I can update it if you insist :)

Reply:
File is not displayed by GitHub, it is better to show it by default:
![image](https://user-images.githubusercontent.com/1002461/74648082-40e91380-517d-11ea-947f-d0adaf2c290e.png)